### ISSUE TITLE
2 pieces of code need to be fixed in colab

### ISSUE THREAD

**Issue by chunshou-Liu** [zhouhaoyi/Informer2020#154]
Hello, your work is excellent, however, two pieces of code need to be fixed in colab. 

1. The checkpoint folder name is Inconsistent while saving and reading 
- @ [Prediction] cell 1
- **The second line need to be changed  to ->** 
`setting = 'informer_ETTh1_ftM_sl96_ll48_pl24_dm512_nh8_el2_dl1_df2048_atprob_fc5_ebtimeF_dtTrue_mxTrue_exp_0'`

```
import os

# set saved model path
setting = 'informer_ETTh1_ftM_sl96_ll48_pl24_dm512_nh8_el2_dl1_df2048_atprob_fc5_ebtimeF_dtTrue_exp_0'
# path = os.path.join(args.checkpoints,setting,'checkpoint.pth')
```

   
2. 'self' needs to be removed from the code
- @[More details about Prediction - prediction function] cell 1
- in this function `def predict(exp, setting, load=False):` 
- **self.args.padding -> args.padding** 

```
# decoder input
        if self.args.padding==0:
            dec_inp = torch.zeros([batch_y.shape[0], exp.args.pred_len, batch_y.shape[-1]]).float()
        elif self.args.padding==1:
            dec_inp = torch.ones([batch_y.shape[0], exp.args.pred_len, batch_y.shape[-1]]).float()
        else:
            dec_inp = torch.zeros([batch_y.shape[0], exp.args.pred_len, batch_y.shape[-1]]).float()
        dec_inp = torch.cat([batch_y[:,:exp.args.label_len,:], dec_inp], dim=1).float().to(exp.device)
```

**Comment by cookieminions** [comment]
Thank you very much for your reminder! We have fixed this error.
